src/ImageSharp/Processing/Processors/Normalization/AdaptiveHistogramEqualizationSlidingWindowProcessor{TPixel}.cs

```diff
@@ -257,7 +257,7 @@ protected override void OnFrameApply(ImageFrame<TPixel> source)
         [MethodImpl(InliningOptions.ShortMethod)]
         private void AddPixelsToHistogram(ref Vector4 greyValuesBase, ref int histogramBase, int luminanceLevels, int length)
         {
-            for (int idx = 0; idx < length; idx++)
+            for (nint idx = 0; idx < length; idx++)
```

Comment:
@JimBobSquarePants could you explain a bit what the advantages of using `nint` is? This is kind of new to me.

Reply:
Basically better codegen. 

https://github.com/SixLabors/ImageSharp/pull/1734#discussion_r687902047